src/python/pants/backend/python/util_rules/interpreter_constraints.py

```diff
@@ -192,7 +192,7 @@ def minimum_python_version(self) -> str | None:
         """
         if self.includes_python2():
             return "2.7"
-        max_expected_py3_patch_version = 15  # The current max is 3.6.12.
+        max_expected_py3_patch_version = 15  # The current max is 3.6.13.
```

Comment:
Can we automate a test for this in case the number is exceeded? They got to 2.7.16 at one point.